apps/server/lib/google-drive/src/internal/to-file.ts

```
@@ -0,0 +1,147 @@
+ * @param {number=} options.lastModified the last modified timestamp
+ * @returns a {@link File} with the given properties
+ */
+export async function toFile(
+	value: ToFileInput | PromiseLike<ToFileInput>,
+	name?: string | null | undefined,
+	options?: FilePropertyBag | undefined
+): Promise<File> {
+	checkFileSupport();
+
+	// If it's a promise, resolve it.
+	value = await value;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)